crates/ruff_python_formatter/src/snapshots/ruff_python_formatter__tests__ruff_test__statement__delete_py.snap

```diff
@@ -0,0 +1,137 @@
+---
+source: crates/ruff_python_formatter/src/lib.rs
+expression: snapshot
+---
+## Input
+```py
+x = 1
+aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa = 1
+b, c, d = (2, 3, 4)
+
+# Delete something
+del x  # Deleted something
+# Done deleting
+
+# Delete something
+del (
+    # Deleting something
+    x  # Deleted something
+    # Finishing deletes
+) # Completed
+# Done deleting
+
+# Delete something
+del (
+    # Deleting something
+    x  # Deleted something
+    # Finishing deletes
+) # Completed
+# Done deleting
+
+# Delete something
+del (
+    # Deleting something
+    x,  # Deleted something
+    # Finishing deletes
+) # Completed
+# Done deleting
+
+# Delete something
+del (
+    # Deleting something
+    x  # Deleted something
+    # Finishing deletes
+
+    # Dangling comment
+) # Completed
+# Done deleting
+
+# Delete something
+del x, aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa, b, c, d  # Delete these
+# Ready to delete
+
+# Delete something
+del (
+    x,
+    # Deleting this
+    aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa,
+    b,
+    c,
+    d
+    # Deleted
+) # Completed
+# Done
+```
+
+
+
+## Output
+```py
+x = 1
+aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa = 1
+b, c, d = (2, 3, 4)
+
+# Delete something
+del x  # Deleted something
+# Done deleting
+
+# Delete something
+del (
+    # Deleting something
+    x  # Deleted something
+    # Finishing deletes
+    # Completed
+)
+# Done deleting
+
+# Delete something
+del (
+    # Deleting something
+    x  # Deleted something
+    # Finishing deletes
+    # Completed
+)
+# Done deleting
+
+# Delete something
+del (
+    # Deleting something
+    x,  # Deleted something
+    # Finishing deletes
+)  # Completed
+# Done deleting
+
+# Delete something
+del (
+    # Deleting something
+    x  # Deleted something
+    # Finishing deletes
+    # Dangling comment
+    # Completed
+)
```

Comment:
I double-checked this. Could be another black bug.

Reply:
if you filed black bugs link them in comments, it's really helpful to be able to read up later why certain decisions were made

Reply:
https://github.com/psf/black/issues/3741